project-keeper/src/main/java/com/exasol/projectkeeper/validators/files/CiBuildWorkflowGenerator.java

```diff
@@ -47,11 +47,12 @@ private FileTemplate createTemplate(final FileTemplateFromResource template, fin
         final List<StepCustomization> customizations = workflow.map(CustomWorkflow::getSteps)
                 .orElseGet(Collections::emptyList);
         final String environmentName = workflow.map(CustomWorkflow::getEnvironment).orElse(null);
-        return new ContentCustomizingTemplate(template, new GitHubWorkflowCustomizer(
+        return new ContentCustomizingTemplate(template, new GitHubWorkflowCustomizer( //
+                javaVersionCustomizer(),
```

Comment:
No update of the requirement tracing  needed?

Reply:
Ah, ok, this is actually a bugfix. So I guess in this case we don't need to update the requirement tracing.